build.gradle

```diff
@@ -169,18 +169,25 @@ task installConformanceTestDeps(type: Exec) {
     commandLine '.venv/bin/pip', 'install', '-r', 'requirements.txt'
 }
 
-task runConformanceTests(type: Exec) {
+task runConformanceTestsV1(type: Exec) {
     dependsOn << assemble
     dependsOn << startAppForConformance
     dependsOn << loadSchoolDataForConformance
     dependsOn << installConformanceTestDeps
+    mustRunAfter test
+    commandLine '.venv/bin/openregister-conformance', '--api-version', '1', '--no-https', '--register', 'school', 'http://localhost:9090/v1/', '--register-domain', 'test.register.gov.uk'
+}
+
+task runConformanceTestsV2(type: Exec) {
+    dependsOn << runConformanceTestsV1
     finalizedBy stopAppForConformance
     mustRunAfter test
```

Comment:
I think this line is redundant as this already depends on `runConformanceTestsV1` which `mustRunAfter test`